pleskdistup/common/src/dpkg.py

```diff
@@ -5,8 +5,12 @@
 import re
 import subprocess
 import typing
+import time
 
 from . import files, util
+from . import log
```

Comment:
NIT: NIT:  I usually put imports from common on one line. However, this is just a nitpick, and you can ignore it if you prefer.